apps/meteor/client/components/Message/MessageBodyRender/BigEmoji.tsx

```diff
@@ -1,18 +1,29 @@
+import { css } from '@rocket.chat/css-in-js';
 import { BigEmoji as ASTBigEmoji } from '@rocket.chat/message-parser';
 import React, { FC } from 'react';
 
 import Emoji from '../../Emoji';
 
 type BigEmojiProps = {
 	value: ASTBigEmoji['value'];
+	disableBigEmoji: boolean;
 };
 
-const BigEmoji: FC<BigEmojiProps> = ({ value }) => (
-	<>
-		{value.map((block, index) => (
-			<Emoji className='big' key={index} emojiHandle={`:${block.value.value}:`} />
-		))}
-	</>
-);
+const BigEmoji: FC<BigEmojiProps> = ({ value, disableBigEmoji }) => {
+	const bigEmojiClass = css`
+		> span {
+			width: ${disableBigEmoji ? '1em' : '44px'};
+			height: ${disableBigEmoji ? '1em' : '44px'};
+		}
+	`;
```

Comment:
do not always use styling on application side, please adjust component on fuselage side

Reply:
This isn't a fuselage component, I am styling the component rendered by the emoji parser... I've fixed the stylesheet used by the parse, but @gabriellsh asked to not rely on external CSS and create the style on react component.